### ISSUE TITLE
Can't add level in editor.

### ISSUE THREAD

**Issue by DDR0** [anura-engine/anura#274]
When making a new level, the level gets its level id set incorrectly and the editor bails. This didn't used to be an issue, but I think we somewhat recently started sanity-checking level ids.

log:
```
CRITICAL: level.cpp:396 ASSERTION FAILED: Level file frogatto:test.cfg has incorrect id: test.cfg

CRITICAL: 
---
CRITICAL: stack trace:
CRITICAL:   ./anura : Level::Level(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, variant)+0x5d4e
CRITICAL:   ./anura : load_level(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0x4d
CRITICAL:   ./anura : LevelRunner::play_cycle()+0x2afb
CRITICAL:   ./anura : LevelRunner::play_level()+0x13a
CRITICAL:   ./anura : main()+0x6d56
CRITICAL:   /lib/x86_64-linux-gnu/libc.so.6 : __libc_start_main()+0xe7
CRITICAL:   ./anura : _start()+0x2a
CRITICAL: ---
INFO: asserts.cpp:85 : ATTEMPTING TO SEND CRASH REPORT...
[1]    19410 abort (core dumped)  ./anura --level=to-nenes-house.cfg --fps
```

**Comment by DDR0** [comment]
Not to be confused with https://github.com/frogatto/frogatto/issues/362, "Can't add lever in editor."

**Comment by DDR0** [comment]
This is still an issue.

**Comment by DDR0** [comment]
Related to #318. Fixed by commit d6a1f6b40f6703693ffc84a88c3e09b3e8834d60.
